downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -0,0 +1,75 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class Main {
+	static MySQLAdaptor dbAdaptor = null;
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void main(String[] args) throws Exception {
+		
+		logger.info("Beginning Download Directory step");
+		String pathToConfig = "src/main/resources/config.properties";
+		Properties props = new Properties();
+		props.load(new FileInputStream(pathToConfig));
+		//TODO: Check stable identifiers db exists
+		//TODO: Describe each functions outputs in documentation
+		//TODO: File existence check and size check
+		//TODO: Configurable runs
+		//TODO: Parallelize executions
+		//TODO: Integration with Perl wrapper
+		
+		//Set up DB adaptor
+		String username = props.getProperty("username");
+		String password = props.getProperty("password");
+		String database = props.getProperty("database");
+		String host = props.getProperty("host");
+		int port = Integer.valueOf(props.getProperty("port"));
+		int releaseNumber = Integer.valueOf(props.getProperty("release"));
+		dbAdaptor = new MySQLAdaptor(host, database, username, password, port);
+		
+		Runtime.getRuntime().exec("mkdir -p " + releaseNumber);
+		
+		//Begin download directory
+		DatabaseDumps.execute(dbAdaptor, releaseNumber, username, password, host, port, database);
```

Comment:
The list of things to execute should be configurable, at least as a list of named steps in a file. Sometimes we only want to run part of the Download Directory process.

Reply:
Yep that will be done soon, I'm thinking of just having a file listing each of the class names, and then the script will only run the steps with the names in the file, or not commented out in the file

Reply:
Yeah, that would be a nice and simple way to do it.